jobs/build.jpl

```diff
@@ -63,6 +63,7 @@ PARALLEL_BUILDS
 @Library('kernelci') _
 import org.kernelci.util.Job
 
+/*
```

Comment:
Please remove the old code in this PR as it's not a draft any more.

Reply:
ok, updated